### ISSUE TITLE
[FRONT-END] Add-Geeft: Interface Update And Bugfix

### ISSUE THREAD

**Issue by vega-dev** [meltingpotteam/geeft#21]
Description: Some modification and fixes in the AddGeeft Activity interface and code
Requires: layout, Spinner, addGeeft activity
Status: Coding / Designing
Time: 3 Hours

**Comment by danybr-dev** [comment]
- we need only number of exptime.
  put in db only the number,not the entire string. "6 days" --> 6
- send category name in lower case.

**Comment by danybr-dev** [comment]
To be added for Milestone 3:
- "Type" field
- "Geolocalization button"

**Comment by vega-dev** [comment]
i've created a New issue only for the "geolocalization button". When it will be closed we need to test it in the addgeeft and then close this issue.
